src/lib/interactionManager/interaction.ts

```diff
@@ -56,6 +56,8 @@ export class Interaction {
     responder?: Identity
   }
 
+  counterparty?: Identity
```

Comment:
Why is this necessary? I thought this is covered by the `participants` member.

Reply:
it is not covered, it can be either the value of `requester` or `responder`, depending on what "our" role is

Reply:
It's always the `responder` in this case, but I get your point. In this case, I think the `getSummary` method should return the `counterparty`'s summary instead of the `requester`'s.